src/themes/components/simpleTitleHeading.ts

```diff
@@ -0,0 +1,13 @@
+import { ComponentStyleConfig } from '@chakra-ui/react';
+
+const SimpleTitleHeading: ComponentStyleConfig = {
+  parts: ['title'],
```

Comment:
Do we not need to style the `icon` in any way (the colour comes to mind)?
If not is it worth just going for [single part component styling](https://chakra-ui.com/docs/styled-system/component-style#styling-single-part-components)?

Reply:
True! I need to set the color to `gray.900`. Thanks!